evap/contributor/tests/test_views.py

```diff
@@ -272,6 +272,24 @@ def test_display_request_buttons(self):
         self.assertEqual(page.body.decode().count("Request changes"), 0)
         self.assertEqual(page.body.decode().count("Request creation of new account"), 2)
 
+    def test_questionnaire_input_hidden_if_options_empty(self):
```

Comment:
If I understand correctly, the test currently does not show that responsibles can see their options for dropout questionnaires. If we change the code to only show previously selected questionnaires, it still passes. However, we want to guarantee that responsibles can themselves select dropout questionnaires.

I think you should have one dropout questionnaire that is only visible to managers, but selected, and one that is not selected, but visible to everyone.